components/Home/Hero/index.module.scss

```diff
@@ -0,0 +1,110 @@
+@use '../../../styles/typography.scss';
```

Comment:
Why this import is needed? 🤔

Reply:
We are extending .t-subheading from this package in line number 25